lib/serviceLocator.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:get_it/get_it.dart';
+import 'package:hoopr/services/challenges.dart';
+import 'package:hoopr/services/db.dart';
+
+GetIt locator = GetIt.instance;
+
+void setupLocator() {
```

Comment:
This is called in `main.dart` before `runApp()`, allows us to access these singletons globally